core/src/test/java/org/testcontainers/junit/ParameterizedDockerfileContainerTest.java

```diff
@@ -26,39 +17,33 @@
 public class ParameterizedDockerfileContainerTest {
 
     @Parameterized.Parameters(name = "{0}")
-    public static Object[] data() {
-        return new Object[] { "alpine:3.2", "alpine:3.3" };
+    public static Object[][] data() {
+        return new Object[][] {
+                { "alpine:3.2", "3.2"},
+                { "alpine:3.3", "3.3"},
+                { "alpine:3.4", "3.4"}
+        };
     }
 
-    public ParameterizedDockerfileContainerTest(String baseImage) {
+    public ParameterizedDockerfileContainerTest(String baseImage, String expectedVersion) {
         container = new GenericContainer(new ImageFromDockerfile().withDockerfileFromBuilder(builder -> {
```

Comment:
`ImageFromDockerfile` is not needed anymore

Reply:
Do you mean that we can just pull the plain container directly?
This test is partly intended to demonstrate that you can build images inside of a parameterised test, so even though there's no customisation, I'm still quite keen to keep it this way.

Reply:
I've added a comment to try and make this clearer - WDYT?

Reply:
@rnorth ah, true! Didn't realize the purpose of this test 😂